wwwroot/inc/snmp.php

```diff
@@ -4310,6 +4320,37 @@
 		'text' => 'HP 1810-8G v2 (J9802A)',
 		'processors' => array ('procurve-8ports-1000T'),
 	),
+	'11.2.3.7.11.151' => array
+	(
+		'dict_key' => 2242,
+		'text' => 'HP 1810-24G v2 (J9803A), 24 x Gigabit Ethernet 10/100/1000, 2 x SFP+',
+		'processors' => array ('procurve-25-to-26-combo-1000SFP','procurve-modular-1000T'),
```

Comment:
Neither of these two processors will match "Port: x Gigabit - Level" or "Port: x SFP - Level". Please see Mantis#1285 for better suggestions.